app/src/ui/diff/side-by-side-diff.tsx

```diff
@@ -302,7 +302,7 @@ export class SideBySideDiff extends React.Component<
     }
 
     const lineNumberWidth = `${getLineWidthFromDigitCount(
-      this.state.diff.maxLineNumberDigitCount
+      this.state.diff.maxLineNumber.toString().length
```

Comment:
From what I read, there are faster ways of getting how many digits a number has. Haven't looked into it with much detail (lack of time today) but in case you want to take a look: https://stackoverflow.com/a/14879700

This applies to other instances where you do the same `.toString().length` dance 😄

Reply:
That seems to be the fastest for positive integers, which we should only ever have.